docs/cheatsheet/select.rst

```diff
@@ -0,0 +1,114 @@
+.. _ref_cheatsheet_select:
+
+Select
+======
+
+Select a Movie with associated actors and reviews with their authors:
```

Comment:
Whenever you use some DB schema (like movies in this case) add a `.. note::` block on that page with a link to it.

Reply:
So in this case it would be a link to the cheatsheet on types I guess, since I literally use these same types in those SDL examples. Is that right?

Reply:
Sure